samples/Dantooine/Dantooine.Client/Server/Startup.cs

```diff
@@ -0,0 +1,174 @@
+               options.Scope.Add("profile");
+               options.Scope.Add("api1");
+               options.SaveTokens = true;
+               options.GetClaimsFromUserInfoEndpoint = true;
+           });
+
+            services.AddControllersWithViews(options =>
+                 options.Filters.Add(new AutoValidateAntiforgeryTokenAttribute()));
+
+            services.AddRazorPages();
+            services.AddHttpForwarder();
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IHttpForwarder forwarder)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseWebAssemblyDebugging();
+            }
+            else
+            {
+                app.UseExceptionHandler("/Error");
+            }
+
+            app.UseSecurityHeaders(GetSecurityHeaderPolicy(env.IsDevelopment(),  "https://localhost:44319"));
+
+            app.UseHttpsRedirection();
+            app.UseBlazorFrameworkFiles();
+            app.UseStaticFiles();
+
+            app.UseRouting();
+            app.UseAuthentication();
+            app.UseAuthorization();
+
+            var httpClient = new HttpMessageInvoker(new SocketsHttpHandler()
+            {
+                UseProxy = false,
+                AllowAutoRedirect = false,
+                AutomaticDecompression = DecompressionMethods.None,
+                UseCookies = false
+            });
+            var transformer = new CookieTokenTransformer(); // or HttpTransformer.Default;
+            var requestConfig = new ForwarderRequestConfig { ActivityTimeout = TimeSpan.FromSeconds(100) };
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapRazorPages();
+                endpoints.MapControllers();
+                endpoints.Map("/api/DantooineApi1", async httpContext =>
+                {
+                    await forwarder.SendAsync(httpContext, "https://localhost:44343/", httpClient, requestConfig, transformer);
+                }).RequireAuthorization();
+
+                endpoints.MapFallbackToPage("/_Host");
+            });
+        }
+
+        public static HeaderPolicyCollection GetSecurityHeaderPolicy(bool isDev, string idpHost)
+        {
+            var policy = new HeaderPolicyCollection()
+                .AddFrameOptionsDeny()
+                .AddXssProtectionBlock()
+                .AddContentTypeOptionsNoSniff()
+                .AddReferrerPolicyStrictOriginWhenCrossOrigin()
+                .AddCrossOriginOpenerPolicy(builder =>
+                {
+                    builder.SameOrigin();
+                })
+                .AddCrossOriginResourcePolicy(builder =>
+                {
+                    builder.SameOrigin();
+                })
+                .AddCrossOriginEmbedderPolicy(builder =>
+                {
+                    builder.RequireCorp();
+                })
```

Comment:
Can we do that instead?

```suggestion
                .AddCrossOriginOpenerPolicy(builder => builder.SameOrigin())
                .AddCrossOriginResourcePolicy(builder => builder.SameOrigin())
                .AddCrossOriginEmbedderPolicy(builder => builder.RequireCorp())
```

Reply:
updated